.github/workflows/build-image.yml

```diff
@@ -0,0 +1,32 @@
+# Triggered by Pull Request or Manually (from GitHub UI) events
+
+name: Publish image to Registry
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Push to Registry
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2.3.1
+
+    - name: Prepare tag
+      id: prepare
+      run: |
+        commit_id=$(git rev-parse --short HEAD)
+        branch_name=${{ github.head_ref }}
+        tag="${branch_name}-${commit_id}"
```

Comment:
am I right that this tag will be `-<commit>` if running locally or in master (not from Pull request)? If so maybe we can try to append the `-` conditionally?

Reply:
added conditional if github head_ref is empty - get github ref